HW04/ivan_shklovski/notification.rb

```diff
@@ -0,0 +1,12 @@
+class Notification
+  attr_reader :homework, :status
+
+  def initialize(homework:, status: nil)
+    @homework = homework
+    @status = status
+  end
+
+  def to_s
+    @status || 'empty notification'
+  end
```

Comment:
You have `attr_reader` let's use it 
```suggestion
  def to_s
    status || 'empty notification'
  end
```